ForgeUnity/Assets/BeardedManStudios/Scripts/Multiplayer Menu/JoinMenu.cs

```diff
@@ -128,7 +146,8 @@ private void LocalServerLocated(NetWorker.BroadcastEndpoints endpoint, NetWorker
 		{
 			MainThreadManager.Run(() =>
 			{
-				AddServer(endpoint.Address, endpoint.Port);
+                if (!endpoint.Address.Equals(LocalNetworkScanning.GetLocalIPAddress()))
```

Comment:
hmm.. wouldn't this prevent listing servers that are running on the same machine as the client?

Reply:
hmmmm, perhaps so. This makes sense for my use case of having each instance start up a host automatically... I'll remove.

Reply:
It is just a convenience for local testing.